plugins/com.google.cloud.tools.eclipse.dataflow.ui/src/com/google/cloud/tools/eclipse/dataflow/ui/launcher/PipelineOptionsSelectionDialog.java

```diff
@@ -140,7 +140,7 @@ public String getText(Object element) {
         return null;
       }
       String simpleName = Signature.getSimpleName(element.toString());
-      return String.format("%s - %s", simpleName, element.toString());
+      return String.format("%s - %s", simpleName, element.toString()); //$NON-NLS-1$
```

Comment:
Thanks.  Are you sure that should be localized in RTL languages? Will an RTL UI truncate and scroll to the left instead of the right? I'm not really familiar with RTL UIs.